pkgs/development/tools/build-managers/msbuild/create-deps.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p msbuild
+set -euo pipefail
+
+cat << EOL
+{ fetchurl }: [
+EOL
+
+tmpdir="$(mktemp -d -p "$(pwd)")" # must be under source root
+trap 'rm -rf $tmpdir' EXIT
+
+(
+    ulimit -n 8192 # https://github.com/NuGet/Home/issues/8571
```

Comment:
Is this only in the subshell? because otherwise it would actually decrease the value on my system and could cause problems.

Reply:
Yeah, I just checked and it doesn't change the parent shell.